ddtrace/tracer/option.go

```diff
@@ -562,6 +588,21 @@ func newConfig(opts ...StartOption) *config {
 	// This allows persisting the initial value of globalTags for future resets and updates.
 	globalTagsOrigin := c.globalTags.cfgOrigin
 	c.initGlobalTags(c.globalTags.get(), globalTagsOrigin)
+
+	// TODO: change the name once APM Platform RFC is approved
```

Comment:
If possible, would be nice to have the actual RFC number here.

Reply:
There is no RFC "number" I believe and I would not be happy to add a link to a private google doc in the code :/